react/src/views/Box/BoxView.tsx

```diff
@@ -83,23 +117,109 @@ const BTBox = () => {
     },
   });
 
-  const [updateBoxLocation, mutationStatus] = useMutation<
+  const [updateNumberOfItemsMutation] = useMutation<
+    UpdateNumberOfItemsMutation,
+    UpdateNumberOfItemsMutationVariables
+  >(UPDATE_NUMBER_OF_ITEMS_IN_BOX_MUTATION, {
+    refetchQueries: [
+      {
+        query: BOX_BY_LABEL_IDENTIFIER_QUERY,
+        variables: {
+          labelIdentifier: labelIdentifier,
+        },
+      },
+    ],
+  });
+
+  const [updateBoxLocation, mutationLocationStatus] = useMutation<
     UpdateLocationOfBoxMutation,
     UpdateLocationOfBoxMutationVariables
   >(UPDATE_LOCATION_OF_BOX_MUTATION);
 
+  const {
+    isOpen: isPlusOpen,
+    onOpen: onPlusOpen,
+    onClose: onPlusClose,
+  } = useDisclosure();
+  const {
+    isOpen: isMinusOpen,
+    onOpen: onMinusOpen,
+    onClose: onMinusClose,
+  } = useDisclosure();
+
   if (loading) {
-    return <div>Loading...</div>;
+    return <APILoadingIndicator />;
   }
-  if (mutationStatus.loading) {
-    return <div>Updating box...</div>;
+  if (mutationLocationStatus.loading) {
+    return <div>Updating number of items...</div>;
   }
-  if (error || mutationStatus.error) {
-    console.error("Error in BoxView: ", error || mutationStatus.error);
+  if (error || mutationLocationStatus.error) {
+    console.error(
+      "Error in BoxView Overlay: ",
+      error || mutationLocationStatus.error
+    );
     return <div>Error!</div>;
   }
 
-  const boxData = mutationStatus.data?.updateBox || data?.box;
+  const boxData = mutationLocationStatus.data?.updateBox || data?.box;
+
+  const onSubmitTakeItemsFromBox = (
+    boxFormValues: ChangeNumberOfItemsBoxData
+  ) => {
+    console.log("boxLabelIdentifier", labelIdentifier);
+    console.log("boxFormValues", boxFormValues);
+
+    if (
+      boxFormValues.numberOfItems &&
+      boxFormValues.numberOfItems > 0 &&
+      boxData?.items
+    ) {
+      updateNumberOfItemsMutation({
```

Comment:
I think in a follow up PR we could try to generalise (e.g. with a generic component or custom hook) the functionality of the "Add" and "Take" features